### DISCUSSION TITLE
TypeError: undefined is not iterable

### DISCUSSION THREAD

**Discussion by kaisiant** [reactivemarkets/react-financial-charts#478] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/26655111/107198865-9f636d80-6a30-11eb-9493-3e6cf9696849.png)

After added component `<OHLCTooltip />`, the error occurred. Not sure what am I did wrong.
CSB: https://codesandbox.io/s/modern-night-6r59c?file=/src/Chart.js

**Comment by markmcdowell** [comment, marked as answer]
Tooltips need to be inside the `<Chart />` component, it's just placed under the wrong component. 

https://codesandbox.io/s/loving-stonebraker-fnbzk?file=/src/Chart.js
